lib/containers/bats.pm

```diff
@@ -68,3 +68,13 @@ sub delegate_controllers {
         systemctl "daemon-reload";
     }
 }
+
+sub enable_modules {
+    my ($version, $sp, $host_distri) = get_os_release;
+    my $arch = get_required_var('ARCH');
+    my $scc_timeout = 1200;
+
+    script_retry("SUSEConnect -p sle-module-desktop-applications/$version/$arch", delay => 60, retry => 3, timeout => $scc_timeout);
```

Comment:
```suggestion
    add_suseconnect_product(get_addon_fullname('desktop'));
    add_suseconnect_product(get_addon_fullname('sdk'));
    add_suseconnect_product(get_addon_fullname('python3'));
```

And of course add the corresponding imports:

```
use registration qw(register_product add_suseconnect_product get_addon_fullname);
```

I think this is cleaner, WDYT?

Reply:
Much better!  There's some code in tests/containers/bci_* that could benefit from this readability change.